lib/modules/browser_console.js

```diff
@@ -26,12 +26,31 @@ class Log {
     }
 }
 
+function stringifyArg(arg) {
```

Comment:
Consider making this "private" (_stringifyArg) as it will only be called in this class